src/mutation/create-user.use-case.ts

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+
+export interface CreateUserInput {
+  name: string;
+  email: string;
+  birthDate: string;
+  password: string;
+}
+
+export async function createUserMutation(input: CreateUserInput) {
+  const response = await axios.post("http://localhost:4001/graphql", {
+    operationName: null,
+    variables: {
+      data: {
+        name: input.name,
+        email: input.email,
+        birthDate: input.birthDate,
+        password: input.password,
+      },
```

Comment:
```suggestion
      data: input,
```

Imagino que o objeto input já seja exatamente o que vc quer passar.